sematic/examples/cifar_classifier/main.py

```diff
@@ -0,0 +1,89 @@
+# Standard Library
+import argparse
+import logging
+
+# Sematic
+import sematic
+from sematic.ee.ray import RayNodeConfig
+from sematic.examples.cifar_classifier.pipeline import pipeline
+from sematic.examples.cifar_classifier.train_eval import (
+    EvaluationConfig,
+    TrainingConfig,
+    TrainLoopConfig,
+)
+from sematic.types.types.aws.s3 import S3Location
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+LOCAL_TRAINING_CONFIG = TrainingConfig(
+    n_workers=1,
+    worker=RayNodeConfig(
+        cpu=2,
+        memory_gb=8,
+        gpu_count=0,
+    ),
+    checkpoint_dir=S3Location.from_uri("s3://sematic-dev/ray-demo"),
```

Comment:
Please make this configurable through an arg parameter, so that users outside the company can run this.

Reply:
Users outside the company are not likely to be able to run this anyway, since it requires a lot of specifics (Kuberay, GPUs, Sematic EE, ray AIR dependency). But I'll make it configurable just in case.

Reply:
Thank you! Please standardize this to a `DEFAULT_*` constant, like in the other PR.

Reply:
done